<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>dialog_null_picLayoutDialog</class>
 <widget class="QDialog" name="dialog_null_picLayoutDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>448</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label_picLayoutDialog_title">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>650</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>画面布局</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_picLayoutDialog_shutdown">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>0</y>
     <width>50</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>X</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_picLayoutDialog_picLayout">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>90</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>画面布局</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_picLayoutDialog_picOne">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>130</y>
     <width>31</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>1</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_picLayoutDialog_picTwo">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>130</y>
     <width>31</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>2</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_picLayoutDialog_picThree">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>130</y>
     <width>31</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>3</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_picLayoutDialog_picFour">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>130</y>
     <width>31</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>4</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_picLayoutDialog_picFive">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>130</y>
     <width>31</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>5</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_picLayoutDialog_picSix">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>130</y>
     <width>31</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>6</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_picLayoutDialog_picSeven">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>130</y>
     <width>31</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>7</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_picLayoutDialog_singelPic">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>130</y>
     <width>67</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>单画面</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_picLayoutDialog_picExchange">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>230</y>
     <width>67</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>画面轮显</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_picLayoutDialog_picExchange">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>230</y>
     <width>86</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_picLayoutDialog_picEight">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>130</y>
     <width>31</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>8</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_picLayoutDialog_dataShow">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>230</y>
     <width>67</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>数据显示</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_picLayoutDialog_dataShow">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>230</y>
     <width>101</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_picLayoutDialog_mulPic">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>180</y>
     <width>51</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>多画面</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_picLayoutDialog_mulPicEight">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>180</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>8画面</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_picLayoutDialog_mulPicFour">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>180</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>4画面</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_picLayoutDialog_mulPicNine">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>180</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>9画面</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_picLayoutDialog_mulPicSixteen">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>180</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>16画面</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_picLayouDialog_use">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>420</y>
     <width>97</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>应用</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_picLayouDialog_sure">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>420</y>
     <width>97</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>确定</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_picLayouDialog_cancel">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>420</y>
     <width>97</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>取消</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
